pkg/controller/openlibertytrace/openlibertytrace_controller.go

```diff
@@ -233,16 +228,25 @@ func (r *ReconcileOpenLibertyTrace) UpdateStatus(conditionType openlibertyv1beta
 	// Keep the old `LastTransitionTime` when pod and status have not changed
```

Comment:
Maybe we can re-use the openlibertyv1beta1.SetOperationCondtion method here

Reply:
There is some additional logic in this method due to the fact that pod could change in CR.